### PR TITLE
Microservices/fix 04 01

### LINKED ISSUE #8593: Typo in First Image comparing patterns
It's written AMPQ, it shouldn't be AMQP (Advanced Message Queuing Protocol) ?

this image:
https://github.com/dotnet/docs/blob/master/docs/standard/microservices-architecture/architect-microservice-container-applications/media/image15.png

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 89fd8427-589c-9158-f9e9-6b59d34ac4e2
* Version Independent ID: 3091e297-8939-cc0c-7437-5ec8be56ae3c
* Content: [Communication in a microservice architecture](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture)
* Content Source: [docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md](https://github.com/dotnet/docs/blob/master/docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md)
* Product: **dotnet**
* GitHub Login: @CESARDELATORRE
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

General contraction fixes

Fixes #8593 (typo in image)

### CODE REVIEW

File: docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ For instance, imagine that your client application makes an HTTP API call to an
...
 
-- Failure in any one microservice. If you implemented a chain of microservices linked by HTTP calls, when any of the microservices fails (and eventually they will fail) the whole chain of microservices will fail. A microservice-based system should be designed to continue to work as well as possible during partial failures. Even if you implement client logic that uses retries with exponential backoff or circuit breaker mechanisms, the more complex the HTTP call chains are, the more complex it is to implement a failure strategy based on HTTP.
+- Failure in any one microservice. If you implemented a chain of microservices linked by HTTP calls, when any of the microservices fails (and eventually they will fail) the whole chain of microservices will fail. A microservice-based system should be designed to continue to work as well as possible during partial failures. Even if you implement client logic that uses retries with exponential backoff or circuit breaker mechanisms, the more complex the HTTP call chains are, the more complex it's to implement a failure strategy based on HTTP.
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md]: The contraction here makes the sentence difficult to parse. It's best to leave it as "it is"

File: docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Client and services can communicate through many different types of communicatio
 
 The first axis is defining if the protocol is synchronous or asynchronous:
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md]: nit unrelated to your PR for lines 24 and 30: is defining --> defines

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @mvelosop. I've left one nit unrelated to your PR, as well as one case in which the contraction makes the sentence more difficult to understand.

File: docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md (hunk 1 above)
Comment (mvelosop) [reply docs/standard/microservices-architecture/architect-microservice-container-applications/distributed-data-management.md]: Oops, fixed.

File: docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md (hunk 2 above)
Comment (mvelosop) [reply docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md]: Fixed, and line 30 too.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @mvelosop. I'll merge your PR now.

**Pull request merged**